core/src/tab/tab.tsx

```diff
@@ -21,57 +21,56 @@ interface TabStyle {
 
 interface Props {
 	children: Tab[];
+	activeTab: string;
+	setActiveTab: (tab: string) => void;
 	initialTab?: string;
 	noPadding?: boolean;
 	style?: TabStyle;
 	fullHeight?: boolean;
+	callbackOnTab?: (tabId: string) => void;
```

Comment:
In which case should we use this?

Reply:
In this case, if we want to callback a function when clicking the tab. Ex, we want to change the query of URL